### PR TITLE
GSoC 2023 

### PR DESCRIPTION
**Pull request by hegner**

### CODE REVIEW

File: gsoc/2022/index.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+---
+title: "Google Summer of Code"
+author: "Andrei Gheata & Benedikt Hegner"
+layout: default
+---
+
+# ![CERN]({{ site.baseurl }}/images/CERN-HSF-GSoC-logo.png){:height="100px"} Google Summer of Code 2023
```
Comment (graeme-a-stewart) [inline gsoc/2022/index.md:7]: Should be 2022

File: gsoc/guideline.md [hunk 2]
```diff
@@ -82,6 +82,6 @@ create a file for your organization and edit its contents as appropriate.
 
 Organization logos should be called `ORGANIZATION-logo.png` and be less than 100kB in size.
 
-## HSF GSoC Mentor Guideline 2022
+## HSF GSoC Mentor Guideline 2023
 
-[2022 HSF GSoC Mentor Guideline](https://docs.google.com/document/d/1wfkPyPmfK6RlV4FCLTEFwdx9FmSSruH5DN4QeGlhHJg/edit?usp=sharing)
+[2022 HSF GSoC Mentor Guideline](https://docs.google.com/document/d/1bEkKj2UtTyumYFT96Uxgbj3F0W_Td-mw1cvO4vlx5d8/edit?usp=sharing)
```
Comment (graeme-a-stewart) [inline gsoc/guideline.md:87]: 2023

File: gsoc/2022/index.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+The CERN software for experiments (CERN EP-SFT) group has participated in the GSoC since 2011. Since 2017 the program has expanded to involve the high-energy physics community under the umbrella of the HEP Software Foundation.
+
+Information from last year’s GSoC can be found [here]({{ site.baseurl }}/gsoc/2021/index.html).
+
+
+### GSoC 2022
+
+In 2022 HSF is participating in the program as a GSoC umbrella organization, under the name CERN-HSF. The HSF project ideas are listed [below](#projects-in-2022). There are important changes in the [program rules](https://summerofcode.withgoogle.com/rules/)  and the [timeline](https://summerofcode.withgoogle.com/how-it-works/) compared to the previous year, described in this news [blog](https://opensource.googleblog.com/2021/11/expanding-google-summer-of-code-in-2022.html) and in the GSoC 2022 [announcement](https://groups.google.com/g/google-summer-of-code-mentors-list/c/zIX5CEOJWZo/m/94V-2F1FBgAJ). Most notably, there are now both 175 hours and 350 hours projects. The maximum duration of projects was now extended to 22 weeks, which gives some more flexibility on the scheduling of work.
```
Comment (graeme-a-stewart) [inline gsoc/2022/index.md:32]: The link to the GSoC 2022 announcement seems to be dead (or only available to subscribers to the list). We can probably just remove it now.

**Review by graeme-a-stewart — CHANGES_REQUESTED** [review]
Great start - I picked up a few places where we still need to change the year.

I also noticed that the text under **Student pages** needs to change (and there are no blogs available, evidently).

**Comment by graeme-a-stewart** [comment]
Also, once this PR is accepted we should change the front page centre bar to highlight GSoC again. Pretty much cut 'n' paste from https://github.com/HSF/hsf.github.io/commit/97f514c7e0239d1911db09c0b837692ea16b8417 will do the trick.

**Comment by hegner** [comment]
Thanks for spotting the wrong years. Fixed.
I also added the announcement

**Review by graeme-a-stewart — APPROVED** [review]

**Pull request merged**